lib/workos/hash_provider.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+# typed: true
+
+module WorkOS
+  # Module to include an explicit method for converting a model into a Hash containing
+  # its attributes. Default implementation will simply call to_json. Individual classes
+  # may override.
+  module HashProvider
+    include Kernel
```

Comment:
Do we need to include `Kernel`?

My understanding is that `Kernel` methods are already available everywhere, and so including this module doesn't do anything.